### DISCUSSION TITLE
Github Copilot not working properly for me

### DISCUSSION THREAD

**Discussion by mrinal18** [community/community#6778] [Copilot Conversations] [answered]
VSCODE Version: 1.61.2 (Universal)
Github copilot version: v1.6.3450

Problem:
initially when i used copilot with comments, it gave me suggestions based on those comments. i am trying in python itself and it seems like copilot isnt responding. for most cases it is not able to generate in copilot window as well even for a function as simple as adding the two numbers
<img width="1117" alt="Screen Shot 2021-10-26 at 9 47 47 AM" src="https://user-images.githubusercontent.com/17683007/138892308-134d19ae-11b2-48e6-a35a-11f0b97447cb.png">

i will only be seeing this for long time. and still it has not synthesized.

Note: i am not using any proxy or firewall.

Thanks.

**Comment by Fire-** [comment]
I have what I think may be the same issue; Home network, no proxy/firewall.

`vscode 1.66.2`
`copilot v1.15.5815`

The output tab for the extension is filled with what looks to be completions, but the actual copilot window never changes from the aforementioned `Synthesizing 0/10 solutions`

```log
[INFO] [default] [2022-05-05T03:24:34.277Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-05-05T03:24:34.379Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 101 ms
[INFO] [streamChoices] [2022-05-05T03:24:34.379Z] solution 5 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.380Z] solution 4 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.380Z] solution 2 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.381Z] solution 6 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.383Z] solution 9 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.384Z] solution 3 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.385Z] solution 7 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.385Z] solution 1 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.385Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
[INFO] [streamChoices] [2022-05-05T03:24:34.385Z] solution 8 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-54MOQB2K2jQ9om7WwHTmi50dRFpNY}] created: [{1651721074}]
```

**Comment by davecheney** [comment, marked as answer]
> Github copilot version: v1.6.3450

Please upgrade

**Reply by tmlewin** [reply]
Hi Dave, I noticed that copilot doesn't auto complete entire functions like before. it can only auto complete one liners. I have to keep hitting tab every line multiple times till the function is autocompleted. Its started happening 2 days ago. back in the day lets says i create a function like function palindrome(){. As soon as i open the first brace, it autocompletes the entire block of code and i hit tab once and i am done. Now i have to keep it htting tab at every line of code it suggests

**Comment by 0xmovses** [comment]
I'm having this issue for neovim. There is only one release so I cannot upgrade. :Copilot status - enabled and online. :Copilot panel Synthesizing 0/10 solutions (Duplicates hidden)

**Reply by hatemets** [reply]
I'm having the same issue. I tried downgrading Node from 19 to 16.15.0, but it didn't work.
